apps/openmw/mwrender/sky.cpp

```diff
@@ -1337,6 +1340,87 @@ class RainFader : public AlphaFader
     osg::Uniform* mRainIntensityUniform;
 };
 
+void SkyManager::setCamera(osg::Camera *camera)
+{
+  mCamera = camera;
```

Comment:
Strangely, the identation here is wrong while it's fine in all of the other added code.